src/csharp/build_packages_dotnetcli.bat

```diff
@@ -19,8 +19,6 @@ set VERSION=1.11.0-dev
 set NUGET=C:\nuget\nuget.exe
```

Comment:
please resolve the conflict in this file.

Reply:
actually the conflict seems to be in build_pacakges_dotnetcli.sh.

Reply:
Done.